setup.py

```diff
@@ -101,6 +101,7 @@
         "pyfai           <= 2023.9.0",  # breaking changes
         "scikit-image   >= 0.19.0, !=0.21.0",  # regression in ellipse fitting"
         "scikit-learn   >= 1.0",
+        "shapely",
```

Comment:
can we pin this to `> 2.0.0`?

Reply:
I did have a look to see if avoiding it was an option but I think this is a better choice.

Reply:
It's fairly established and light weight so I'm not too concerned about it.  It just isn't quite as fast as I like it to be.